### PR TITLE
add helper for run examples

### PR DESCRIPTION
**Pull request by ilyar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,55 @@ Each new interface must create a folder, which has a readme.md desciption of an
...
+Interface not implemented in cli-browser
+
+**in web-browser**
+```shell
+src=../CountryInput/examples bash ../script/debot-run.sh 
+src=../DateTimeInput/examples bash ../script/debot-run.sh
+src=../EncryptionBoxInput/examples bash ../script/debot-run.sh
+src=../Media/examples bash ../script/debot-run.sh
+src=../QRCode/examples bash ../script/debot-run.sh
+src=../Sdk/examples bash ../script/debot-run.sh
```
Comment (Keshoid) [inline README.md:67]: Sdk is a DEngine built-in interface, not a browser interface

File: README.md [hunk 2]
```diff
@@ -15,3 +15,55 @@ Each new interface must create a folder, which has a readme.md desciption of an
...
+
+```shell
+cd mkdir tmp && cd tmp
+```
+
+#### Run
+
+Compile, deploy and run examples
+
+**in cli-browser `everos-cli`**
```
Comment (Keshoid) [inline README.md:42]: Why do you split interfaces? `cli` and `not cli`.

File: README.md [hunk 3]
```diff
@@ -15,3 +15,55 @@ Each new interface must create a folder, which has a readme.md desciption of an
...
+
+```shell
+src=../AddressInput/examples bash ../script/debot-run.sh
+src=../AmountInput/examples bash ../script/debot-run.sh
+src=../Base64/examples bash ../script/debot-run.sh
+src=../ConfirmInput/examples bash ../script/debot-run.sh
+src=../Hex/examples bash ../script/debot-run.sh
+src=../Json/examples bash ../script/debot-run.sh
+src=../Menu/examples bash ../script/debot-run.sh
+src=../Network/examples bash ../script/debot-run.sh
```
Comment (Keshoid) [inline README.md:52]: Base64, Hex, Json, Network are built-in interafces in DEngine.

File: README.md [hunk 4]
```diff
@@ -15,3 +15,55 @@ Each new interface must create a folder, which has a readme.md desciption of an
...
 * Your browser should clearly state interfaces which it supports (by version number as stated in this repository).
+
+
+### Run examples
+
+#### Setup
+
+```shell
+npm install --global everdev
+npx everdev sol update
```
Comment (Keshoid) [inline README.md:26]: it works without npx. I think, you can delete it everywhere

File: script/debot-run.sh [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+addressWallet() {
+  addressContract "$(artifactContract "${2:-SurfMultisigWallet}")" "${1:-$signer}"
+}
+
+artifactContract() {
+  walletName="${1:-SurfMultisigWallet}" # SafeMultisigWallet | SetcodeMultisigWallet
+  slug="$(echo "${walletName::-6}" | tr '[:upper:]' '[:lower:]')/${walletName}"
+  rev="5adb4c394fcc22b790ede9a3c18e74f1db8034c8"
+  path="tonlabs/ton-labs-contracts/${rev}/solidity"
```
Comment (Keshoid) [inline script/debot-run.sh:32]: Switch to https://github.com/EverSurf/contracts/tree/main/surfmultisig

File: script/debot-run.sh [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+fi
+
+printf "Deploy ABI... "
+appABI=$(< "${appArtifact}.abi.json" jq --compact-output | xxd -ps -c 20000)
+npx everdev contract run --address "${appAddr}" "${appArtifact}" setABI --input "dabi:$appABI" &>>build.log
+echo "✓"
+
+echo "DeBot ${appAddr}"
+echo "npx tonos-cli debot --debug fetch ${appAddr}"
+npx tonos-cli debot --debug fetch "${appAddr}"
```
Comment (Keshoid) [inline script/debot-run.sh:97]: It's better to run without `--debug`.

**Review by Keshoid — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -15,3 +15,55 @@ Each new interface must create a folder, which has a readme.md desciption of an
...
+```
+
+#### Run
+
+Compile, deploy and run examples
+
+**in cli-browser `everos-cli`**
+
+```shell
+src=../AddressInput/examples bash ../script/debot-run.sh
```
Comment (Keshoid) [inline README.md:45]: What is `current dir` for running this script? I suppose we should run it from repo root dir.

File: README.md [hunk 8]
```diff
@@ -15,3 +15,55 @@ Each new interface must create a folder, which has a readme.md desciption of an
...
+#### Setup
+
+```shell
+npm install --global everdev
+npx everdev sol update
+npx everdev tonos-cli update
+npx everdev network default se
+npx everdev signer default coder
+npx everdev signer add giver 172af540e43a524763dd53b26a066d472a97c4de37d5498170564510608250c3
+npx everdev network giver se 0:b5e9240fc2d2f1ff8cbb1d1dee7fb7cae155e5f6320e585fcc685698994a19a5 --signer giver
```
Comment (Keshoid) [inline README.md:31]: Failed to run this cmd. everdev 1.1.1

File: README.md [hunk 9]
```diff
@@ -15,3 +15,55 @@ Each new interface must create a folder, which has a readme.md desciption of an
...
+### Run examples
+
+#### Setup
+
+```shell
+npm install --global everdev
+npx everdev sol update
+npx everdev tonos-cli update
+npx everdev network default se
+npx everdev signer default coder
```
Comment (Keshoid) [inline README.md:29]: What is coder?

**Review by Keshoid — CHANGES_REQUESTED** [review]

File: script/debot-run.sh (hunk 6 above)
Comment (ilyar) [reply script/debot-run.sh:97]: nice try)))

**Pull request closed without merging**